Update scenario.md (#475)
Avaiga/taipy-doc: docs/manuals/core/concepts/scenario.md

Introducing the `Scenario`, a fundamental key concept in Taipy.

> A Taipy scenario represents a business problem with consistent data and parameters.

Reviewer: I would remove 'Taipy' here. Maybe use italics for the first occurrence of the term 'scenario'.

> It serves as a powerful tool to create different versions of a problem by incorporating assumptions. This flexibility is particularly valuable for impact analysis and decision-making processes, enabling users to create, store, edit, and execute multiple scenarios within the same application.

Reviewer 2: I don't like this line. What do you think about the following

```suggestion
Scenarios are a powerful tool to create different versions of a problem under different assumptions. This is particularly valuable for what-if analysis in decision-making processes, enabling users to create, store, edit, and execute multiple scenarios with various parameters within the same application.
```

> It serves as a powerful tool to create different versions of a problem by incorporating assumptions. This flexibility is particularly valuable for impact analysis and decision-making processes, enabling users to create, store, edit, and execute multiple scenarios within the same application.

Reviewer: 'particularly' -> 'especially'.
Although correct, I personally seldom see that.

> Once an initial scenario has been analyzed, users can modify the input data nodes (excluding the intermediates and output data nodes), rerun his pipelines, and compare results.

Reviewer 2: ```suggestion
Once an initial scenario has been created and analyzed, users may want to rerun their pipelines with different input data and compare results.
```

Reviewer: Introducing 'pipeline' here may be confusing: scenarios at this point have not been describes as holding pipelines...

> This involves instantiating a second scenario, executing it, and comparing the outcomes with the first scenario.

Reviewer 2: ```suggestion
This involves instantiating a second scenario, changing the input data, executing it, and comparing the outcomes with the first scenario.
```

    ![scenarios](../pic/scenarios.svg){ align=left }

Two use-cases arise from the utilization of Taipy scenarios:

### - Use case 1 : 

> Each _scenario_ represents a distinct instance of a business problem. 

Reviewer: The "new" convention is to use stars (\*) to mark italics.
That's for a clearer support of Python symbols that can contain underscores, but not stars.

    Then the end-user creates another scenario for February using the new information provided for the
    February period. And so on.

### - Use case 2:

> Two _scenarios_ represent the same business problem instance, but with different sets of assumptions.

Reviewer: Same here : \*scenarios\*

!!! example

>     The end-user wants to simulate how our capacity data affects production planning for the February situation.

Reviewer: "how our capacity" -> "how the capacity"?
